enhancements/sig-policy/59-ansible-always-mode/README.md

```diff
@@ -0,0 +1,176 @@
+# Add an "always" Ansible Job Run Mode to the Policy addon
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+The Open Cluster Management (OCM) Policy addon Ansible integration should support a new run mode of `always` so that an
+Ansible job can be run for every unique policy violation per managed cluster.
+
+## Motivation
+
+The Open Cluster Management (OCM) Policy addon supports running Ansible jobs on policy violations. The Policy addon
+currently supports the run modes `once` and `disabled`. The `once` mode is limiting since after a policy violation
+occurs, the Ansible job runs once and then it is disabled. A user must manually renable it for the Ansible job to run
+again on the next policy violation.
+
+An additional mode of `always` should be added so that an Ansible job can be run for every unique policy violation per
+managed cluster. A simple example is that a user might use the OCM Policy addon Ansible integration to create tickets in
+their ticketing system on every policy violation per cluster. The new `always` mode would enable this use-case.
```

Comment:
I'm wondering if `always` is the right word for this - it's not like this automation will be continuously running under this proposal. Something like `onEveryViolation` might be more accurate.

And it opens the question of: what if a user wants to do something when a policy becomes compliant on a cluster? Eg, if an automation should open a ticket when something is noncompliant, maybe I want another automation to close a ticket when it becomes compliant?

Reply:
We agreed on `everyEvent`.

Running automation on other states is a good idea but out of scope of this enhancement since it's handled by the `eventHook` field on `PolicyAutomation` objects.